xpublish-exploring/demo.py

```diff
@@ -15,9 +15,14 @@ def get_datasets(self):
     @hookimpl
     def get_dataset(self, dataset_id: str):
         try:
-            return xr.tutorial.open_dataset(dataset_id)
+            ds = xr.tutorial.open_dataset(dataset_id)
+            if ds.cf.coords['longitude'].dims[0] == 'longitude':
+                ds = ds.assign_coords(longitude=(((ds.longitude + 180) % 360) - 180)).sortby('longitude')
+                # TODO: Yeah this should not be assumed... but for regular grids we will viz with rioxarray so for now we will assume
+                ds = ds.rio.write_crs(4326)
+            return ds
         except HTTPError:
-            return None
+            return ds
```

Comment:
`HTTPError` is thrown when the dataset can't be accessed. This can happen either due to a server error, or because it doesn't exist. In this case we want to return `None`, so that Xpublish can continue asking other dataset providers if they have the given `dataset_id`